modules/drivers/mongo/src/metabase/driver/mongo/query_processor.clj

```diff
@@ -460,16 +460,28 @@
 (defmethod ->rvalue :/
   [[_ & [_ & divisors :as args]]]
   (let [build-division (fn build-division
-                         [[dividend & divisors]]
-                         (if (seq divisors)
-                           {"$divide" [(build-division divisors) (->rvalue dividend)]}
-                           (->rvalue dividend)))
+                         [[head & tail]]
+                         (if (seq tail)
+                           {"$divide" [(build-division tail) (->rvalue head)]}
+                           (->rvalue head)))
+        ;; args go in reversed because build-division works outside in (/ 1 2 3) => (/ (/ 1 2) 3)
```

Comment:
Isn't this the shape a straight `reduce` would build?

Reply:
Thank you, I thought so but was having trouble making the leap.